packages/feature-flags/src/prompts/enterNewEnvironmentName.js

```diff
@@ -0,0 +1,24 @@
+const inquirer = require('inquirer');
+
+module.exports = async function newFlagName(existingEnvironments) {
```

Comment:
:nit: Since this both asks for the new environment's name _and_ gathers which env should be the sourceEnvironment, what do you think about calling this `setupNewEnvironment.js`?

Also :nit:, function name here is misnamed (probably copy/pasta'd)

Reply:
Good catch ❤️